content-repo/create-docs.sh

```diff
@@ -169,28 +169,20 @@ cp ${CONTENT_GIT_DIR}/Tests/demistomock/demistomock.py .
 cp ${CONTENT_GIT_DIR}/Tests/Marketplace/approved_tags.json .
 cp ${CONTENT_GIT_DIR}/Tests/Marketplace/approved_usecases.json .
 
-ARTICLES_DIR=${SCRIPT_DIR}/extra-docs/articles
-DEMISTO_CLASS_OVERVIEW="All Python integrations and scripts have available as part of the runtime the \`demisto\` class object. The object exposes a series of API methods which are used to retrieve and send data to the Cortex XSOAR Server.
-
-:::note
-The \`demisto\` class is a low level API. For many operations we provide a simpler and more robust API as part of the  [Common Server Functions](https://xsoar.pan.dev/docs/integrations/code-conventions#common-server-functions).
-:::"
-DEMISTO_CLASS_DOCS_CMD=("./gen_pydocs.py" "-d" "${ARTICLES_DIR}" "-i" "demisto-class" "-t" "'Demisto Class'" "-m" "demisto" "-p" "demisto." "-o" "'${DEMISTO_CLASS_OVERVIEW}'")
-mv demistomock.py demisto.py
+echo "$(tail -n +6 CommonServerPython.py)" > CommonServerPython.py
```

Comment:
Why are we doing this?

Reply:
Anyway add a comment explaining this.

Reply:
this is to remove the first few lines https://github.com/demisto/content/blob/master/Packs/Base/Scripts/CommonServerPython/CommonServerPython.py#L1-L5
ill add a comment